bandit/plugins/logging_config_insecure_listen.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding:utf-8 -*-
+#
+# Copyright 2014 Hewlett-Packard Development Company, L.P.
```

Comment:
I assume this is a copy and paste from another file. If you really do work for HP, then you should denote 2022 for the year. Otherwise, the copyright corresponds to you as the code author.

Reply:
Yes, it was a copy paste mistake :) thanks for the comment, I updated it.